packages/postgres-database/src/simcore_postgres_database/migration/versions/e8057a4a7bb0_new_services_tags_table.py

```diff
@@ -0,0 +1,44 @@
+"""new services_tags table
+
+Revision ID: e8057a4a7bb0
+Revises: 7604e65e2f83
+Create Date: 2024-08-23 12:12:32.883771+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e8057a4a7bb0"
+down_revision = "7604e65e2f83"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "services_tags",
```

Comment:
Note: I believe we discussed in a previous PR (proposed by @GitHK, which I liked) that when we have a many-to-many relationship, it should be named services_to_tags. However, when it is an extension of a resource, such as access rights for a resource, then projects_access_rights would be a more appropriate name. But this was just a discussion.

Reply:
- IMO the `to` does not bring any extra info here. 
- `services_tags`, `projects_tags` sounds reasonable to me. As you say, it is a sub-resource of a service and therefore the mapping is implicit.
- Yes, based on that logic it should be `services_to_groups` instead of `services_access_rights`  as we have just corrected. 

Please come to a conclusion so we can simply align

Reply:
to be honest. I prefer services_access_rights than services_to_groups... but well

Reply:
Sorry for the late response. My view is that tags_access_rights represents a one-to-many relationship, while services_to_tags is automatically without much thinking many-to-many, thanks to the "to" in its name. Therefore, all instances of resource_to_groups should gradually be renamed to resource_access_rights, as it also represents usually one-to-many relationship, now I am talking about the "access rights" tables.